ruby/.rubocop.yml

```diff
@@ -162,4 +165,3 @@ Style/TrailingCommaInHashLiteral:
 
 Style/WordArray:
   Enabled: false
-  
```

Comment:
Is this intentional?

Reply:
my editorconfig must have kicked in. It was an extra blank line